app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,12 @@
+class DashboardController < ApplicationController
+  def index
+    @user_count = User.count
+    @question_count = Question.count
+    @answer_count = Answer.count
+
+# despite documentation saying Tenants.left_outer_joins should work, it doesn't
+# I inspected the methods of Tenant and verified there was no such method.
+    @tenants = Tenant.joins("left outer join tenant_stats on tenants.id = tenant_stats.tenant_id").select("name", "ifnull(request_count, 0) request_count").all
```

Comment:
left_outer_joins was added in Rails 5. No reason we couldn't have used Rails 5 for this project but rails version was locked to 4.2.4 ( see the [Gemfile](https://github.com/justjohn2001/babel/blob/master/Gemfile)). Feel free to update that in the Gemfile and re-bundle if you'd like to use the latest and greatest